src/net/sourceforge/kolmafia/request/RelayRequest.java

```diff
@@ -1240,7 +1251,7 @@ private boolean sendHardcorePVPWarning(final String urlString) {
     }
 
     // Don't remind a second time in a session if you decide not to do it.
-    if (urlString.contains(CONFIRM_HARDCOREPVP)) {
+    if (urlString.contains(Confirm.HARDCOREPVP.toString())) {
```

Comment:
Curious why it is necessary to explicitly call toString here.

Reply:
`Confirm` doesn't implement `CharSequence`, so it has to be converted to a string. I could implement it, but I thought it was simpler to just convert it.